morf-core/src/main/java/org/alfasoftware/morf/metadata/SchemaResource.java

```diff
@@ -48,4 +49,13 @@ public interface SchemaResource extends Schema, AutoCloseable {
   default Optional<DatabaseMetaDataProvider> getDatabaseMetaDataProvider() {
     return Optional.empty();
   }

```

Comment:
Hopefully, we would be able to drop this `getDatabaseMetaDataProvider()` method :)

Reply:
Removed.